MainRepository/Program.cs

```diff
@@ -1,7 +1,24 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿//Program to find the prime number or not.
+Console.Write ("Enter the Number: ");
+string number;
+number = Console.ReadLine ();
+Prime (number);
+static void Prime (string number) {
+   int result;
+   if (int.TryParse (number, out result)) {
+      int divisors = 0;
+      for (int i = 1; i <= result; i++) {
+         if (result % i == 0)
+            divisors++;
+      }
+      if (divisors == 2)

```

Comment:
The reason for the check is there will be always 1 and the prime number. You can start i with 2 and end till result - 1. Then when you find result % i == 0, then it is not prime. You can return immediately instead of going through the entire number. Consider the input is 2222222222, you will figure out the number is not prime when i = 2, but i will increment up to the input to print the result.